integration/run-tests.sh

```diff
@@ -2,5 +2,5 @@
 
 pushd ..
 CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -tags release -o piper
-go test -tags=integration -timeout 20m ./integration/...
+go test -tags=integration -timeout 20m -run TestNPMProject ./integration/...
```

Comment:
Needs to be reverted before merging, correct?

Reply:
yes, would be nice to add an option to run a single test case here..